[2024-06-18T07:35:58.962+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T07:35:58.992+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: full_dbt_execute.run_dbt manual__2024-06-18T07:35:56.808011+00:00 [queued]>
[2024-06-18T07:35:58.997+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: full_dbt_execute.run_dbt manual__2024-06-18T07:35:56.808011+00:00 [queued]>
[2024-06-18T07:35:58.997+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-18T07:35:59.007+0000] {taskinstance.py:2330} INFO - Executing <Task(DockerOperator): run_dbt> on 2024-06-18 07:35:56.808011+00:00
[2024-06-18T07:35:59.015+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=70) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T07:35:59.017+0000] {standard_task_runner.py:63} INFO - Started process 72 to run task
[2024-06-18T07:35:59.016+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'full_dbt_execute', 'run_dbt', 'manual__2024-06-18T07:35:56.808011+00:00', '--job-id', '221', '--raw', '--subdir', 'DAGS_FOLDER/full_dbt_execute.py', '--cfg-path', '/tmp/tmpe6lvith8']
[2024-06-18T07:35:59.019+0000] {standard_task_runner.py:91} INFO - Job 221: Subtask run_dbt
[2024-06-18T07:35:59.068+0000] {task_command.py:426} INFO - Running <TaskInstance: full_dbt_execute.run_dbt manual__2024-06-18T07:35:56.808011+00:00 [running]> on host b1fbcf01fc69
[2024-06-18T07:35:59.192+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='full_dbt_execute' AIRFLOW_CTX_TASK_ID='run_dbt' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T07:35:56.808011+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-18T07:35:56.808011+00:00'
[2024-06-18T07:35:59.196+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T07:35:59.257+0000] {docker.py:366} INFO - Starting docker container from image custom_dbt_image
[2024-06-18T07:35:59.275+0000] {docker.py:374} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-06-18T07:36:01.846+0000] {docker.py:436} INFO - [0m07:36:01  Running with dbt=1.5.0
[2024-06-18T07:36:01.990+0000] {docker.py:436} INFO - [0m07:36:01  Found 3 models, 4 tests, 0 snapshots, 0 analyses, 341 macros, 0 operations, 0 seed files, 2 sources, 0 exposures, 0 metrics, 0 groups
[2024-06-18T07:36:01.991+0000] {docker.py:436} INFO - [0m07:36:01
[2024-06-18T07:36:02.447+0000] {docker.py:436} INFO - [0m07:36:02  Concurrency: 1 threads (target='dev')
[2024-06-18T07:36:02.448+0000] {docker.py:436} INFO - [0m07:36:02
[2024-06-18T07:36:02.450+0000] {docker.py:436} INFO - [0m07:36:02  1 of 3 START sql table model mint.my_first_dbt_model ........................... [RUN]
[2024-06-18T07:36:02.603+0000] {docker.py:436} INFO - [0m07:36:02  1 of 3 OK created sql table model mint.my_first_dbt_model ...................... [[32mOK[0m in 0.15s]
[2024-06-18T07:36:02.605+0000] {docker.py:436} INFO - [0m07:36:02  2 of 3 START sql table model mint.stage_subscriptions .......................... [RUN]
[2024-06-18T07:36:02.663+0000] {docker.py:436} INFO - [0m07:36:02  2 of 3 OK created sql table model mint.stage_subscriptions ..................... [[32mOK[0m in 0.06s]
[2024-06-18T07:36:02.664+0000] {docker.py:436} INFO - [0m07:36:02  3 of 3 START sql view model mint.my_second_dbt_model ........................... [RUN]
[2024-06-18T07:36:02.710+0000] {docker.py:436} INFO - [0m07:36:02  3 of 3 OK created sql view model mint.my_second_dbt_model ...................... [[32mOK[0m in 0.05s]
[2024-06-18T07:36:02.754+0000] {docker.py:436} INFO - [0m07:36:02
[2024-06-18T07:36:02.755+0000] {docker.py:436} INFO - [0m07:36:02  Finished running 2 table models, 1 view model in 0 hours 0 minutes and 0.76 seconds (0.76s).
[2024-06-18T07:36:02.772+0000] {docker.py:436} INFO - [0m07:36:02
[2024-06-18T07:36:02.773+0000] {docker.py:436} INFO - [0m07:36:02  [32mCompleted successfully[0m
[2024-06-18T07:36:02.775+0000] {docker.py:436} INFO - [0m07:36:02
[2024-06-18T07:36:02.775+0000] {docker.py:436} INFO - [0m07:36:02  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2024-06-18T07:36:04.032+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T07:36:04.131+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=full_dbt_execute, task_id=run_dbt, run_id=manual__2024-06-18T07:35:56.808011+00:00, execution_date=20240618T073556, start_date=20240618T073558, end_date=20240618T073604
[2024-06-18T07:36:04.161+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T07:36:04.205+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-18T07:36:04.207+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
